python/lsst/afw/image/exposure/_exposureContinued.py

```diff
@@ -94,6 +96,25 @@ def setVariance(self, variance):
 
     variance = property(getVariance, setVariance)
 
+    def getConvexPolygon(self):
+        """Get the convex polygon associated with the bounding box corners.
+
+        Returns
+        -------
+        convexPolygon : `lsst.sphgeom.ConvexPolygon`
+            Returns `None` if exposure does not have a valid WCS.
+        """
+        wcs = self.wcs
+        if wcs is None:
+            return None
+
+        bbox = lsst.geom.Box2D(self.getBBox())
+        corners = [wcs.pixelToSky(corner).getVector()
+                   for corner in bbox.getCorners()]
+        return lsst.sphgeom.ConvexPolygon(corners)
```

Comment:
Since you are not padding this there is a chance that because distortions and such that the spherical 'rectangle" created by this will actually not contain pixels from the edges of the bbox. Is this something we should worry about, or just put a message? In the past we have dilated such that you can be sure that all points in the image are in the region, but the converse may not be true. This requires 2 checks, but you can be sure no sph points that are rejected are not actually in the image.